test/extended/olm/olmv1.go

```diff
@@ -714,3 +729,38 @@ spec:
 		o.Expect(err).NotTo(o.HaveOccurred(), "Job failed or timed out")
 	}
 }
+
+func waitForDefaultCatalogs(ctx context.Context, oc *exutil.CLI) {
+	var defaultCatalogs = []string{
+		"openshift-certified-operators",
+		"openshift-community-operators",
+		"openshift-redhat-marketplace",
+		"openshift-redhat-operators",
+	}
+	for _, cat := range defaultCatalogs {
+		g.By(fmt.Sprintf("checking that %q is exists and is serving content", cat))
+		err := wait.PollUntilContextTimeout(ctx, 5*time.Second, 2*time.Minute, true, func(ctx context.Context) (bool, error) {
```

Comment:
It would be good to know how many iterations occur here, so that we can see if we're waiting on catalogs at all for these tests.